host/src/types/uuid.rs

```diff
@@ -1,7 +1,7 @@
 use crate::codec::{Decode, Encode, Error, Type};
 
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Eq, Hash, Clone, Copy)]
```

Comment:
I'd rather keep the uuid not Copy if possible, as it can easily explode memory usage if copied uncritically throughout the usage.